backend/app/verification.py

```diff
@@ -252,6 +268,17 @@ class VerificationStatus(BaseModel):
     detail: str | None = None
 
 
+class ArchiveRequest(BaseModel):
+    endoflife: str | None = None
```

Comment:
I would like this as `endoflife_message`, as I always get confused

Reply:
Also not sure, why this can be `None` might be a nice thing to force having this.